doc/en/how-to/output.rst

```diff
@@ -549,6 +549,27 @@ captured output:
     By default, parametrized variants of skipped tests are grouped together if
     they share the same skip reason. You can use ``--no-fold-skipped`` to print each skipped test separately.
 
+
+.. _truncation-params:
+Modifying truncation limits
+--------------------------------------------------
+
+.. versionadded: 8.4
+
+Default truncation limits are 8 lines or 640 characters, whichever comes first.
+To set custom truncation limits you can use following ``pytest.ini`` file options:
+
+.. code-block:: ini
+
+    [pytest]
+    truncation_limit_lines = 10
```

Comment:
Ahh sorry I forgot, we also need to explicitly document these options in [reference.rst](https://github.com/pytest-dev/pytest/blob/main/doc/en/reference/reference.rst)... see the `confval` definitions at the bottom. Please add one section for each of these options, in alphabetical order. :+1:

Reply:
(that's why the readthedocs build failed)